app/Providers.tsx

```diff
@@ -0,0 +1,31 @@
+"use client";
+
+import { ThemeProvider } from "next-themes";
+import { ReactNode } from "react";
+
+// import { ClientReload } from "@/components/ClientReload";
+
+interface ClientLayoutWrapperProps {
+  children: ReactNode;
+  useClientReload: boolean;
+  defaultTheme: string;
+}
+
+/**
+ * A client component to provide the theming providers and wrappers we use.
+ */
+const Providers = ({
+  children,
+  // useClientReload,
+  defaultTheme,
+}: ClientLayoutWrapperProps) => {
+  return (
+    <ThemeProvider attribute="class" defaultTheme={defaultTheme}>
+      {/*TODO: Might not need client reloading*/}
+      {/*{useClientReload ? <ClientReload /> : null}*/}
```

Comment:
Won't need this anymore since contentlayer properly hooks into Next's fast reload system.

Need to update the `start` command to just be `next` and remove the `next-remote-watch` script, and remove this.